_opensearch/rest-api/document-apis/index-document.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: default
+title: Index document
+parent: Document APIs
+grand_parent: REST API reference
+nav_order: 1
+---
+
+# Index document
+
+Introduced 1.0
+{: .label .label-purple}
+
+Before you can search for data, you must first add a document by indexing it.
+
+## Example
+
+```json
+GET sample-index/_doc/1
+```
+
+## Path and HTTP methods
+
+```
+PUT <index>/_doc/<_id>
+POST <index>/_doc
+
+PUT <index>/_create/<_id>
+POST <index>/_create/<_id>
+```
+
+## URL parameters
+
+In your request, you must specify the index you want to add your document to. If the index doesn't already exist, OpenSearch automatically creates the index and adds in your document. All other URL parameters are optional.
+
+Parameter | Type | Description | Required
+:--- | :--- | :--- | :---
+&lt;index&gt; | String | Name of the index. | Yes
+&lt;_id&gt; | String | A unique identifier to attach to the document. To automatically generate an ID, use `POST <target/_doc` in your request. | No
+if_seq_no | Integer | Only perform the operation if the document has the specified sequence number. | No
```

Comment:
Missing '>.' You could also explicitly say that to automatically generate an ID, use POST instead of PUT.